cpp/src/examples/babyllama/baby_llama_handler.cc

```diff
@@ -0,0 +1,269 @@
+#include "src/examples/babyllama/baby_llama_handler.hh"
+
+#include <typeinfo>
+
+namespace llm {
+
+std::pair<std::shared_ptr<torch::jit::script::Module>,
+          std::shared_ptr<torch::Device>>
+LlmHandler::LoadModel(
+    std::shared_ptr<torchserve::LoadModelRequest>& load_model_request) {
+  try {
+    auto device = GetTorchDevice(load_model_request);
+    // Load dummy model
+    auto module = std::make_shared<torch::jit::script::Module>(
+        torch::jit::load(fmt::format("{}/{}", load_model_request->model_dir,
+                                     manifest_->GetModel().serialized_file),
+                         *device));
+
+    char checkpoint_path[] = "/home/ubuntu/serve/cpp/stories15M.bin";
+    build_transformer(&transformer, checkpoint_path);
+
+    char tokenizer_path[] =
+        "/home/ubuntu/serve/cpp/src/examples/image_classifier/babyllama/"
```

Comment:
Path is hard coded at present -- read from config file

Reply:
Updated the code to read the tokenizer and model path from a config file